test/fake-timers-test.js

```diff
@@ -53,6 +53,69 @@ const utilPromisifyAvailable = promisePresent && utilPromisify;
 const timeoutResult = global.setTimeout(NOOP, 0);
 const addTimerReturnsObject = typeof timeoutResult === "object";
 
+describe("issue #2449: perminent loss of native functions", () => {
```

Comment:
```suggestion
describe("issue #2449: permanent loss of native functions", function () {
```
typo-fix. Also Mocha relies heavily on `this`, so it's a malpractice to use arrow functions. Not sure why we do not have ESLint rules to handle this 🤔

Reply:
Done. Also, I forgot to run `npm run lint`, so I didn't see the problem with the arrow functions. However, since I'm not using `this` in my tests, it wouldn't have mattered.